js/import/rules.import.js

```diff
@@ -52,7 +52,36 @@ function buildSelector(mapping, basePath) {
   if (mapping.xpath) {
     return xPathToCss(mapping.xpath.replace(basePath, ''));
   }
-  return '';
+  return undefined;
+}
+
+/**
+ * Build a block cells object from a list of mappings.
+ * @param mappingList
+ */
+function buildBlockCellsFromMapping(mappingList = []) {
```

Comment:
This is the new function that converts an array of mapping objects into a compatible `cells` object.